FormType/NewThreadMessageFormType.php

```diff
@@ -16,20 +18,48 @@ class NewThreadMessageFormType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('recipient', 'fos_user_username', array('label' => 'recipient', 'translation_domain' => 'FOSMessageBundle'))
-            ->add('subject', 'text', array('label' => 'subject', 'translation_domain' => 'FOSMessageBundle'))
-            ->add('body', 'textarea', array('label' => 'body', 'translation_domain' => 'FOSMessageBundle'));
+            ->add('recipient', LegacyFormHelper::isLegacy() ? 'fos_user_username' : 'FOS\UserBundle\Form\Type\UsernameFormType', array(
+                'label' => 'recipient',
+                'translation_domain' => 'FOSMessageBundle'
+            ))
+            ->add('subject', LegacyFormHelper::getType('Symfony\Component\Form\Extension\Core\Type\TextType'), array(
+                'label' => 'subject',
+                'translation_domain' => 'FOSMessageBundle'
+            ))
+            ->add('body', LegacyFormHelper::getType('Symfony\Component\Form\Extension\Core\Type\TextareaType'), array(
```

Comment:
It looks like i was a bit tired yesterday ^^ i thought about the line 21 of the same file.